app/src/androidTest/java/org/mozilla/focus/activity/DownloadFileTest.kt

```diff
@@ -116,7 +121,7 @@ class DownloadFileTest {
     @Test
     fun openDownloadFileTest() {
```

Comment:
Should we maybe rename this as downloadAndOpenJpgFileTest to match with the others?

Reply:
Done!